src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
+const MAX_HEALTH = 100;
+const DECREASED_HEALTH_BY_BITE = 50;
+
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = MAX_HEALTH;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(animalName) {
```

Comment:
Try to make up a better name. It will be instance of some class, not name string